### PR TITLE
Trim passed array of detected orphans

### PR DESCRIPTION
**Pull request by mloviska**
- Verification run: http://kepler.suse.cz/tests/16800#step/orphaned_packages_check/9

### CODE REVIEW

File: tests/console/orphaned_packages_check.pm [hunk 1]
```diff
@@ -35,14 +33,18 @@ sub compare_orphans_lists {
...
-    my @missed_orphans = map { s/^\s+|\s+$//gr } @{$args{zypper_orphans}};
+    my @detected_orphans = map { s/^\s+|\s+$//gr } @{$args{zypper_orphans}};
+    delete $args{zypper_orphans};
+    my @missed_orphans = @detected_orphans;
+
     if ($args{whitelist}) {
         # Remove duplicate packages from the whitelist
         my %wl = map { $_ => 1 } (split(',', $args{whitelist}));
-        @missed_orphans = grep { !$wl{$_} } @{$args{zypper_orphans}};
+        @missed_orphans = grep { !$wl{$_} } @detected_orphans;
```
Comment (asdil12) [inline tests/console/orphaned_packages_check.pm:43]:
```suggestion
        @missed_orphans = grep { !$wl{$_} } @missed_orphans;
```

Comment (asdil12) [reply tests/console/orphaned_packages_check.pm:43]: Better filter from the var that we already assigned to.

**Review by asdil12 — CHANGES_REQUESTED** [review]

File: tests/console/orphaned_packages_check.pm (hunk 1 above)
Comment (mloviska) [reply tests/console/orphaned_packages_check.pm]: Updated

**Review by asdil12 — APPROVED** [review]
LGTM

**Pull request merged**